erpnext/controllers/subcontracting_controller.py

```diff
@@ -395,6 +395,24 @@ def __get_materials_from_bom(self, item_code, bom_no, exploded_item=0):
 			frappe.get_all("BOM", fields=fields, filters=filters, order_by=f"`tab{doctype}`.`idx`") or []
 		)
 
+	def __get_supplied_items_from_sco(self, sco, item_code):
+		fields =  [
+			"rm_item_code",
+			"rate",
+			"stock_uom",
+			"reserve_warehouse",
+			"stock_uom",
+			"sourced_by_supplier",
+			"required_qty",
+			"sourced_by_hyper",
+			"required_qty as qty_consumed_per_unit"
+		]
+
+		return (
+			frappe.get_all("Subcontracting Order Supplied Item", fields=fields, filters={"parent": sco, "main_item_code":item_code, "sourced_by_supplier": 0}) or []
```

Comment:
`frappe.get_all` will return an empty array by default, do we need the `or []` at the end?